explorer/site-config/anvil-catalog/dev/index/studiesEntityConfig.ts

```diff
@@ -1,26 +1,48 @@
 import { AnVILCatalogStudy } from "../../../../app/apis/catalog/anvil-catalog/common/entities";
-import { buildAnVILCatalogStudies } from "../../../../app/apis/catalog/anvil-catalog/common/utils";
+import { getDbGapId } from "../../../../app/apis/catalog/anvil-catalog/common/transformers";
 import * as Components from "../../../../app/components";
 import {
   ComponentConfig,
   EntityConfig,
   ListConfig,
 } from "../../../../app/config/common/entities";
 import * as ViewBuilder from "../../../../app/viewModelBuilders/anvil-catalog/common/viewModelBuilders";
-import { SOURCE_FIELD_KEY, SOURCE_FIELD_TYPE } from "../../tsv-config";
+import { mainColumn } from "../detail/study/overviewMainColumn";
+import { sideColumn } from "../detail/study/overviewSideColumn";
+import { top } from "../detail/study/top";
 
 /**
  * Entity config object responsible to config anything related to the /explore/studies route.
  */
-export const studiesEntity: EntityConfig<AnVILCatalogStudy> = {
+export const studiesEntityConfig: EntityConfig<AnVILCatalogStudy> = {
   detail: {
     staticLoad: true,
```

Comment:
Insert `detailOverviews: ["Overview"],` to the property `detail`. See `explorer/site-config/anvil/dev/index/datasetsEntityConfig.ts` line 20. This will provide the correct styles for a detailed page.

Reply:
@frano-m, what do you think about making detailOverviews required instead of optional and setting it to [] if there is no overview? This forces it to be specified and makes omitting the style less likely, perhaps?

Reply:
Sounds like a solid plan! Awesome suggestion.

Reply:
Nice, thanks!